scene/main/viewport.cpp

```diff
@@ -2451,6 +2451,14 @@ void Viewport::_gui_update_mouse_over() {
 		return;
 	}
 
+	if (gui.sending_mouse_enter_exit_notifications) {
+		// If notifications are already being sent, delay call to next frame.
+		if (get_tree() && !get_tree()->is_connected("process_frame", callable_mp(this, &Viewport::_gui_update_mouse_over))) {
```

Comment:
This looks like it's updated relatively often, I'd suggest using `SNAME` to avoid bottleneck with `StringName` construction, like so:
```suggestion
		if (get_tree() && !get_tree()->is_connected(SNAME("process_frame"), callable_mp(this, &Viewport::_gui_update_mouse_over))) {
```